installer/install.sh

```diff
@@ -15,39 +16,34 @@ GIT_BRANCH=${GITHUB_HEAD_REF:-`git branch --show-current`}
 echo "Set VAULT_TOKEN in a secret for vault-secrets-operator..."
 # The namespace may not exist already, but don't error if it does.
 kubectl create ns vault-secrets-operator || true
+#kubectl create secret generic vault-secrets-operator \
+#  --namespace vault-secrets-operator \
+#  --from-literal=VAULT_TOKEN=$VAULT_TOKEN \
+#  --from-literal=VAULT_TOKEN_LEASE_DURATION=31536000 \
+#  --dry-run -o yaml | kubectl apply -f -
 kubectl create secret generic vault-secrets-operator \
   --namespace vault-secrets-operator \
-  --from-literal=VAULT_TOKEN=$VAULT_TOKEN \
-  --from-literal=VAULT_TOKEN_LEASE_DURATION=31536000 \
+  --from-literal=VAULT_ROLE_ID=$(vault read --format=json auth/approle/role/rubin-data-dev.slac.stanford.edu/role-id | jq -M .data.role_id  | sed 's/"//g') \
+  --from-literal=VAULT_SECRET_ID=$(vault write -f --format=json auth/approle/role/rubin-data-dev.slac.stanford.edu/secret-id | jq .data.secret_id | sed 's/"//g') \
+  --from-literal=VAULT_TOKEN_MAX_TTL=600 \
```

Comment:
These changes are SLAC-specific and need to be reverted or made conditional in some way.